src/cljs_tooling/util/analysis.cljc

```diff
@@ -137,10 +137,29 @@
         excludes (:excludes (find-ns env ns))]
     (apply dissoc macros excludes)))
 
+(def ^:private common-keywords
+  #{:require :require-macros :import
+    :refer :refer-macros :include-macros
+    :refer-clojure :exclude
+    :keys :strs :syms
+    :as :or
+    :pre :post
+    :let :when :while
+    :clj :cljs
+    :default
+    :else
+    :private :doc :author
+    :gen-class
+    :keywordize-keys
+    :req :req-un :opt :opt-un
+    :args :ret :fn
+    :const
+    :arglists :tag :static :added})
+
 (defn keyword-constants
   "Returns a list of keyword constants in the environment."
   [env]
-  (filter keyword? (keys (:cljs.analyzer/constant-table env))))
+  (concat common-keywords (filter keyword? (keys (:cljs.analyzer/constant-table env)))))
```

Comment:
Why isn't the analyzer returning those? Is tis some bug or something?

Reply:
I'll update the docstring, about the why...well this is a question for David but I guess these keywords are not part of the compiler state, are just used and then discarded...

I took the list from lumo, which is in any case arbitrary.

Reply:
Well, I guess someone should ask him what's the proper way to get those, so we don't have to hardcode them. :-)